### DISCUSSION TITLE
More problems than answers...

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#142] [Redshift Models]
From "The First Year of JWST Day 1 - Session 1 (Sep 11, 2023)"
Space Telescope Science Institute (STScI) Research Staff Channel

https://youtu.be/pAPeLRl5rT0?t=2242 - Haojing Yan
"Last year when we first published our work on very [high redshift z>11 in SMACS 0723](https://dx.doi.org/10.3847/2041-8213/aca80c) we were very badly smashed."

Yan *et al.* used a stricter selection filter, but z_photometric > 13 galaxies still don't show the expected rapid evolution.

![Yan-moderate-evolution](https://github.com/a-cosmology-group/acg/assets/16748402/7f39eeec-333e-45a2-b6c4-ab0dcc244c78)

**Comment by bgreenman** [comment]
Have there been any detailed analysis of these distant galazies

**Reply by RedshiftDrift** [reply]
The talks presented here [https://www.youtube.com/@stsciresearch6722/streams](https://www.youtube.com/@stsciresearch6722/streams) seem to be reporting the most recent analyses.

**Comment by ExpEarth** [comment]
The speaker shown there said in his talk that their data is consistent with the epoch of reionization being at _z_ = 17.3.  Can we take from that that we can expect to see no galaxies with redshifts higher than this?

**Reply by bgreenman** [reply]
I think there might be galaxies beyond z=17.3, if the telescope is programmed to look

**Reply by RedshiftDrift** [reply]
Non-ionized hydrogen gas only blocks UV wavelengths.  Galaxies can be observed beyond $z > 20$ with JWST and radio telescopes, independently of the beginning of the *"period of reionization"*.
Look at the three examples of [F277 dropouts shown here](https://youtu.be/pAPeLRl5rT0?t=2534), those are likely [galaxies at $z \simeq 24$](https://youtu.be/pAPeLRl5rT0?t=2334) (the animation goes to $z = 29.9$).

In any case, the *"period of reionization"* is a theoretical event that happens after the Big Bang, therefore there is no such *"period of reionization"*.

**Reply by bgreenman** [reply]
I was referring to,whether the JWST cornsortium will allow researchers to look beyond z=17.3. I think if such research is allowed we might find interesting galaxies at z=29.9

**Reply by HanDeBruijn** [reply]
@bgreenman 
>I think if such research is allowed we might find interesting galaxies at z=29.9

In a [previous post](https://github.com/orgs/a-cosmology-group/discussions/135#discussioncomment-6956930) an explanation of the numbers $w$ and $W$ is given. If [Halton Arp's Ideas](https://github.com/orgs/a-cosmology-group/discussions/141#discussioncomment-6966770) are worked out _in a proper manner_, then there results _a maximum for the cosmological redshifts that can ever be observed_. That maximum turns out to be given by

$$
1+z_{max} = e^{H/c.W.c/H} = e^W = \exp(3.383634283) \quad \Longrightarrow \quad \Large \boxed{ z_{max} \approx 28.5 }
$$

**Reply by bgreenman** [reply]
Thats a great correction, the explanation shown in the YouTube presentation is slightly off

**Reply by RedshiftDrift** [reply]
@bgreenman the value $z = 29.9$ presented in the YouTube video is only the limit on this specific animation.
By no means should this be mis-interpreted as a limit on redshift!

**Reply by bgreenman** [reply]
correct
